.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] [REQUIRED] Please add `test.md` file after line 2 because `test.md` file will ignore like `node_modules/`!

![image](https://user-images.githubusercontent.com/63915024/192002949-3c8ac6eb-9e0b-4371-a38f-a9c62ddcf3f7.png)